tailwind.config.js

```diff
@@ -1,11 +1,8 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: [
-    "./pages/**/*.{js,ts,jsx,tsx}",
-    "./components/**/*.{js,ts,jsx,tsx}",
-  ],
-  theme: {
-    extend: {},
-  },
-  plugins: [],
-}
\ No newline at end of file
+    content: [
+        "./pages/**/*.{js,ts,jsx,tsx}",
+        "./components/**/*.{js,ts,jsx,tsx}",
+    ],
+    presets: [require("./tailwind.configs/elegant.tailwind.config.js")],
```

Comment:
so they can change this line to reference whichever config they want and they're chilling. allows us to easily mess w fonts and colors in each theme.